modules/atoms/remark/Text.module.css

```
@@ -0,0 +1,16 @@
+.list {
+  list-style: square;
+  color: var(--font-color-default);
+  padding: 1rem 2rem;
+}
+
+.paragraph {
+  font-size: 16px;
+  color: var(--font-color-default);
+  line-height: 1.5rem;
```

adding vertical margins on the paragraphs allows to remove the `<br/>` tags in `about.mdx`, and it will restore the style that we had before with the `my-4` class (currently you can see a difference with master in the paragraph `Qu'est ce que le LyonJS ?`)
```suggestion
  line-height: 1.5rem;
  margin-block: 1rem;
```